client/src/components/WebAuthn/Register.js

```diff
@@ -233,37 +190,78 @@ class Register extends Component {
    * @return {HTMLElement}
    */
   renderActions() {
+    const { ss: { i18n } } = window;
     const { view } = this.state;
     let actions = [];
 
     switch (view) {
       case VIEWS.FAILURE:
         actions = [
-          { action: this.handleStartRegistration, name: 'Retry' },
-          { action: this.handleBack, name: 'Back' },
+          {
+            action: this.handleStartRegistration,
+            name: i18n._t('MFAWebAuthnRegister.RETRY', fallbacks['MFAWebAuthnRegister.RETRY'])
+          },
+          {
+            action: this.handleBack,
+            name: i18n._t('MFAWebAuthnRegister.BACK', fallbacks['MFAWebAuthnRegister.BACK'])
+          },
         ];
         break;
       case VIEWS.READY:
         actions = [
-          { action: this.handleStartRegistration, name: 'Register key' },
-          { action: this.handleBack, name: 'Back' },
+          {
+            action: this.handleStartRegistration,
+            name: i18n._t('MFAWebAuthnRegister.REGISTER', fallbacks['MFAWebAuthnRegister.REGISTER'])
+          },
+          {
+            action: this.handleBack,
+            name: i18n._t('MFAWebAuthnRegister.BACK', fallbacks['MFAWebAuthnRegister.BACK'])
+          },
         ];
         break;
       case VIEWS.REGISTERING:
         actions = [
-          { action: this.handleStartRegistration, name: 'Registering', disabled: true },
-          { action: this.handleBack, name: 'Back', disabled: true },
+          {
+            action: this.handleStartRegistration,
+            name: i18n._t(
+              'MFAWebAuthnRegister.REGISTERING',
+              fallbacks['MFAWebAuthnRegister.REGISTERING']
+            ),
+            disabled: true
+          },
+          {
+            action: this.handleBack,
```

Comment:
Just noticed that `btn-link` as the default for "not primary" button doesn't match the other parts of the MFA UI - can you change it to `btn-secondary` please?

Reply:
Noticed this too while testing it: https://github.com/silverstripe/silverstripe-mfa/pull/196